step/step.go

```diff
@@ -54,15 +56,33 @@ func (r *ReportGenerator) ProcessConfig() (*Config, error) {
 	}
 
 	stepconf.Print(input)
-	r.logger.Println()
 	r.logger.EnableDebugLog(input.Verbose)
 
+	patterns := strings.Split(input.XcresultPatterns, "\n")
+	var filteredPatterns []string
+
+	for _, pattern := range patterns {
+		if pattern == "" {
```

Comment:
NOTE: in some steps, we guard a bit better this sort of input values, things like: trimming spaces before and after the original value, when checking items we also trim spaces and then do the comparism.

Reply:
Thanks for the tip. I have modified this and also the user input splitting above this.